### PR TITLE
fix upgrade step from PR #31.

### LINKED ISSUE #38: V8 Upgrade fails
PR https://github.com/collective/collective.lazysizes/pull/31 leads to an unexpected but interesting error. For any request - including when applying the upgrade step - I get this error:
```(Pdb++) 2017-09-29 15:30:47 ERROR plone.transformchain Unexpected error whilst trying to apply transform chain
Traceback (most recent call last):
  File "/home/_thet/repos-config/dotfiles-thet/dot.buildout/eggs/plone.transformchain-1.2.1-py2.7.egg/plone/transformchain/transformer.py", line 49, in __call__
    newResult = handler.transformIterable(result, encoding)
  File "/home/_thet/data/dev/plone/buildout.coredev/src/collective.lazysizes/src/collective/lazysizes/transform.py", line 160, in transformIterable
    settings = registry.forInterface(ILazySizesSettings)
  File "/home/_thet/repos-config/dotfiles-thet/dot.buildout/eggs/plone.registry-1.1.2-py2.7.egg/plone/registry/registry.py", line 82, in forInterface
    name
KeyError: 'Interface `collective.lazysizes.interfaces.ILazySizesSettings` defines a field `lazyload_authenticated`, for which there is no record.'
```
So, applying the upgrade step fails, leading to a broken site, because the transform fail.

This brings up other questions too:

- plone.transformchains are applied in the ZMI too - shouldn't be the case.
- transaction.commits() fail, when the transformchain fails? - not sure, if that should be the case.

Before thinking how to explain how to fix that, I'll provide a quick fix...

### PR DESCRIPTION
**Pull request by thet**
Fixes #38

### CODE REVIEW

File: src/collective/lazysizes/interfaces.py [hunk 1]
```diff
@@ -31,6 +31,6 @@ class ILazySizesSettings(model.Schema):
             u'&lt;img&gt; and &lt;iframe&gt; elements with that class directly applied to them, or to a parent element, will be skipped.'
         ),
         required=False,
-        default=set([]),
+        default=set(),
```
Comment (thet) [inline src/collective/lazysizes/interfaces.py]: That's not part of the fix, but ``set() == set([])``

File: src/collective/lazysizes/transform.py [hunk 2]
```diff
@@ -157,19 +157,22 @@ def transformUnicode(self, result, encoding):
...
                 return  # transform not enabled for authenticated users
 
         result = self._parse(result)
         if result is None:
             return
 
-        blacklist = self._blacklist(result, settings.css_class_blacklist)
+        blacklist = self._blacklist(
+            result,
+            getattr(settings, 'css_class_blacklist', set())
```
Comment (thet) [inline src/collective/lazysizes/transform.py]: Also not strictly necessary, but I use the same pattern to access the settings like above...

**Comment by hvelarde** [comment]
I can make a release after merging this, or wait until you provide the i18n update and translations, let me know!

**Comment by thet** [comment]
@hvelarde added https://github.com/collective/collective.lazysizes/pull/41

**Pull request merged**